src/mainwindow.cpp

```diff
@@ -383,7 +383,7 @@ void MainWindow::displayTranslation()
     // If window mode is notification, send a notification including the translation result
     const AppSettings settings;
     if (this->isHidden() && settings.windowMode() == AppSettings::Notification)
-        m_trayIcon->showMessage(tr("Translation Result"), ui->translationEdit->toPlainText(), QSystemTrayIcon::NoIcon, 3000);
+        m_trayIcon->showMessage(tr("Translation Result"), ui->translationEdit->toPlainText(), QSystemTrayIcon::NoIcon, m_translationNotificationDisplayTime*1000);
```

Comment:
Could you add a helper method to `m_trayIcon` called `showTranslationMessage` and call it with the translation text?
This will allow to move `m_translationNotificationDisplayTime` to this class and set it from settings by `setTranslationNotificationDisplayTime`.

Also what do you think about to rename `m_translationNotificationDisplayTime` to `m_translationNotificationTimeout`? To make it shorter and as the parameter name in [showMessage](https://doc.qt.io/qt-5.9/qsystemtrayicon.html#showMessage).

Reply:
That'll be better，accept all.

Reply:
> Also what do you think about to rename m_translationNotificationDisplayTime to m_translationNotificationTimeout? To make it shorter and as the parameter name in showMessage.
Do you want me to change all `displayTimes` to `timeout`? Or is it just for `m_translationNotificationDisplayTime`.

Reply:
> Do you want me to change all displayTimes to timeout?

Oh, yes, just forgot about other places :) I like `displayTimes` better, but the whole name becomes is too long in the code, I would suggest shortening it to `timeout`.

Reply:
OK, Updated. The new update includes ensuring **tray icon** display in **Notifcation Mode**.